typescript/ecs/ecs-service-with-logging/index.ts

```diff
@@ -0,0 +1,42 @@
+import ec2 = require('@aws-cdk/aws-ec2');
+import { InstanceType } from '@aws-cdk/aws-ec2';
+import ecs = require('@aws-cdk/aws-ecs');
+import cdk = require('@aws-cdk/cdk');
+
+class WillkommenECS extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const vpc = new ec2.VpcNetwork(this, 'MyVpc', { maxAZs: 2 });
+    const cluster = new ecs.Cluster(this, 'Ec2Cluster', { vpc });
+    cluster.addDefaultAutoScalingGroupCapacity({
+      instanceType: new InstanceType("t2.xlarge"),
+      instanceCount: 3,
+    });
+
+    // create a task definition with CloudWatch Logs
+    const logging = new ecs.AwsLogDriver(this, "AppLogging", {
+        streamPrefix: "myapp",
+    })
+    const taskDef = new ecs.TaskDefinition(this, "MyTaskDefinition", {
+        compatibility: ecs.Compatibility.Ec2,
+    })
+    taskDef.addContainer("AppContainer", {
+        image: ecs.ContainerImage.fromDockerHub("amazon/amazon-ecs-sample"),
```

Comment:
NOOP: Just noting that `fromDockerHub` will also be deprecated in the next release due to https://github.com/awslabs/aws-cdk/commit/11ed69186e80f3667e78f36f94e126a0fb8e9230

Reply:
Its going to be `fromInternet` now right? Can I switch to that or do I have to wait, or should I just leave this as is?

Reply:
In fact it will be `fromRegistry`. No point in changing it before the release though. But if the release happens to occur before this is merged, I am going to ask you to change it after the release.

All in all, it probably makes most sense to hold off on this change for a couple of days, and then change it.